src/bsl/scopeProvider.ts

```
@@ -1,83 +1,48 @@
-import { editor, Position } from 'monaco-editor-core';
-import tokensProvider, { TokensSequence } from './tokensProvider'
-import { Scope, Symbol, GlobalScope, EditorScope, } from '@/scope';
+import { editor } from 'monaco-editor-core';
+import { Scope, Symbol, GlobalScope, EditorScope, TypeDefinition, } from '@/scope';
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)